angular-ngrx-scss/src/app/repository/components/repo-header/repo-heading/repo-heading.component.scss

```diff
@@ -0,0 +1,35 @@
+@use '../../../../shared/styles/variables.scss';
+
+.heading {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+
+  &__container {
+    margin-bottom: 0.25rem;
+    display: flex;
+    gap: 0.5rem;
+    align-items: center;
+
+    a {
+      font-size: 1.475rem;
+      color: variables.$blue600;
+      text-decoration: none;
+    }
+
+    span {
+      color: variables.$gray600;
+      font-size: 1.475rem;
+    }
+  }
+
+  &__privacy {
```

Comment:
Let's not forget to account for the privacy states ("public" and "private") when we implement both.